api/handlers/ethereum2/beacon_node/beacon_node_handler.go

```diff
@@ -151,3 +172,158 @@ func ValidateBeaconNodeExist(c *fiber.Ctx) error {
 	c.Locals("node", node)
 	return c.Next()
 }
+
+// Stats returns a websocket that emits peer  count and node syncing status
+func Stats(c *websocket.Conn) {
+	defer c.Close()
+
+	name := c.Params("name")
+	beaconnode := &ethereum2v1alpha1.BeaconNode{}
+	nameSpacedName := types.NamespacedName{
+		Namespace: c.Locals("namespace").(string),
+		Name:      name,
+	}
+	err := k8sClient.Get(context.Background(), nameSpacedName, beaconnode)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			c.WriteJSON(fiber.Map{
+				"error": fmt.Sprintf("peer by name %s doesn't exist", name),
+			})
+			return
+		}
+		c.WriteJSON(fiber.Map{
+			"error": err.Error(),
+		})
+		return
+	}
+
+	if !beaconnode.Spec.REST {
+		c.WriteJSON(fiber.Map{
+			"error": "node rest is not enabled",
+		})
+		return
+	}
+
+	for {
+		jobs := make(chan request, 2)
+		results := make(chan result, 2)
+
+		for i := 0; i < 2; i++ {
+			go worker(jobs, results)
+		}
+
+		baseUrl := fmt.Sprintf("http://%s.%s:%d/eth/v1/node/", beaconnode.Name, nameSpacedName.Namespace, beaconnode.Spec.RESTPort)
+		jobs <- request{name: "peers", url: fmt.Sprintf("%speer_count", baseUrl)}
+		jobs <- request{name: "isSyncing", url: fmt.Sprintf("%ssyncing", baseUrl)}
+
+		close(jobs)
+
+		type NodePeerCountDto struct {
+			Disconnected  int `json:"disconnected"`
+			Connecting    int `json:"connecting"`
+			Connected     int `json:"connected"`
+			Disconnecting int `json:"disconnecting"`
+		}
+		type NodeSyncingStatusDto struct {
+			HeadSlot     int  `json:"head_slot"`
+			SyncDistance int  `json:"sync_distance"`
+			IsSyncing    bool `json:"is_syncing"`
+			IsOptimistic bool `json:"is_optimistic"`
+		}
+		var nodeStatResponseDto struct {
+			Peers   NodePeerCountDto     `json:"peers"`
+			Syncing NodeSyncingStatusDto `json:"syncing"`
```

Comment:
Similar to `Ethereum` and `Polkadot`, we want to keep the response structure flat (no need for nesting using data and separate structs for every request), this is an example from polkadot stats:
```json
// polkadot stat
{"currentBlock":67502,"highestBlock":17050472,"peersCount":40,"syncing":true}
// ethereum stat
{"currentBlock":"0","highestBlock":"0","peersCount":1}
```

Update the response structure to {currentSlot, targetSlot, peersCount, syncing}

Where:
- currentSlot is NodeSyncingStatusDto.HeadSlot
- targetSlot is NodeSyncingStatusDto.HeadSlot + NodeSyncingStatusDto.SyncDistance
- frontend can divide currentSlot by targetSlot to get sync %
- peersCount is NodePeerCountDto.Connected
- syncing is NodeSyncingStatusDto.IsSyncing

Ethereum beacon chain has a slot of blocks, so we want to show user slots stats
[slot1: b1, b2, b3] [slot2: b4, b5, b6] ...